app/site/controllers.py

```diff
@@ -27,17 +27,16 @@ def index():
       - site
     responses:
       404:
-        description: Publiser does not exist
+        description: Publisher does not exist
       200:
-        description: Succesfuly loaded home page
+        description: Successfully loaded home page
     """
-    user, exception = get_user_from_cookie()
-    if user:
+    if g.jwt_exception:
```

Comment:
We don't want to check for g.jwt_exception on every request. This should be somewhere common (e.g. before request handler or something like that -- i'm not quite sure how flask supports this).

**More important**: why are we handling jwt_exception and redirecting to logout? Why can't we just proceed as normal -- the user just won't be logged in!